draft-ietf-trans-rfc6962-bis.md

```diff
@@ -681,6 +678,30 @@ value will not be a valid X.509v3 signature that could be used in conjunction
 with the TBSCertificate (from `SignedData.encapContentInfo.eContent`) to
 construct a valid certificate.
 
+### Binding Intent to Issue  {#binding_intent_to_issue}
+
+Under normal circumstances, there will be a short time period between
+precertificate submission and issuance of the corresponding certificate.
+Occasional delays are to be expected though (e.g., due to log server downtime),
+and in some cases the CA might not actually issue the corresponding certificate.
+Nevertheless, a precertificate's `signature` indicates the CA's binding intent
+to issue the corresponding certificate, which means that:
+
+* Misissuance of a precertificate is considered equivalent to misissuance of
+  the corresponding certificate. The CA should expect to be held to account,
+  even if the corresponding certificate has not actually been issued.
+
+* Upon observing a precertificate, a client can reasonably presume that the
+  corresponding certificate has been issued. A client may wish to obtain status
+  information (e.g., by using the Online Certificate Status Protocol [RFC6960]
+  or by checking a Certificate Revocation List [RFC5280]) about a certificate
+  that is presumed to exist, especially if there is evidence or suspicion that
+  the corresponding precertificate was misissued.
+
+* TLS clients may have policies that require CAs to be able to revoke, and to
```

Comment:
I admit I find this paragraph a bit confusing - maybe drop "to be able to revoke, and" and just leave the bit about providing certificate status?

Reply:
I think it's important to explicitly state "to be able to revoke", to echo the second of these two points from https://wiki.mozilla.org/CA/Required_or_Recommended_Practices#Precertificates:

> * A CA must provide OCSP services and responses in accordance with Mozilla policy for all certificates presumed to exist based on the presence of a Precertificate, even if the certificate does not actually exist
> * A CA must be able to revoke a certificate presumed to exist, if revocation of the certificate is required under Mozilla policy, even if the certificate does not actually exist.

I don't want CAs to think that they can refuse to revoke a precertificate just because the corresponding certificate hasn't actually been issued.

Reply:
Acknowledged ,good point.